src/io/flutter/inspector/FlutterWidget.java

```diff
@@ -160,7 +160,8 @@ private void init() {
       try {
         // Local copy of: https://github.com/flutter/website/tree/master/_data/catalog/widget.json
         final URL resource = getClass().getResource("widgets.json");
-        final String content = new String(Files.readAllBytes(Paths.get(resource.toURI())));
+        final byte[] contentBytes = ByteStreams.toByteArray((InputStream)resource.getContent());
```

Comment:
DBC: Perhaps add a comment about why we're using resource.getContent() instead of toUri()? This would reduce the chance of re-introducing this issue, here or elsewhere.